### PR TITLE
add script for checking the health of experiments

### PR DESCRIPTION
**Pull request by a-solovyev12**
This PR adds logic for checking the health of experiments when running indexing. We expect a certain number of entries per accession. If for a specific experiment this number is lower than a defined minimum, failure occurs.

### CODE REVIEW

**Comment by suhaibMo** [comment]
Great work @a-solovyev12. Some minor improvements. I would like to see the CI going through GitHub actions to see other tests defined previously are succeeding.

File: .github/workflows/run_ci_tests.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: CI
```
Comment (suhaibMo) [inline .github/workflows/run_ci_tests.yml]: Maybe CI name can be more explicit to the test done here like prefixing INDEX_GXA_CI or anything you feel more self explanatory.

**Comment by suhaibMo** [comment]
In regards to this "We expect a certain number of entries per accession. If for a specific experiment this number is lower than a defined minimum, failure occurs" 

How do we estimate minimum entries for a study?

**Comment by a-solovyev12** [comment]
Thanks for review @suhaibMo! I think the lower limit for the number of entries is somewhat arbitrary and depends on a particular experiment. We might need to change this parameter a couple of times before we find an optimal number.

**Comment by suhaibMo** [comment]
@a-solovyev12 Great all tests passed. I'm wondering the last test added to bats `Check the health of experiments` isn't executed in the github actions as last succeeded was `loading data to Solr` https://github.com/ebi-gene-expression-group/index-gxa/pull/5/checks?check_run_id=1455018156#step:3:212 I'm I missing something?

**Comment by suhaibMo** [comment]
My bad. It is here `Check the health of experiments` https://github.com/ebi-gene-expression-group/index-gxa/runs/1455018156?check_suite_focus=true#step:3:205

Good work @a-solovyev12

**Review by suhaibMo — APPROVED** [review]
LGTM

**Pull request merged**